ui/tests/integration/task-log-test.js

```diff
@@ -61,6 +61,7 @@ module('Integration | Component | task log', function(hooks) {
   });
 
   hooks.afterEach(function() {
+    window.localStorage.clear();
```

Comment:
This is a tad sad computer. Not something to change now, but I get it. Have you looked at ember-window-mock at all?

Reply:
Not in earnest.